File: MlbDataServer/Contracts/GameCenter.cs
using System;
using System.Xml.Serialization;

namespace MlbDataServer.Contracts
{
	[Serializable, XmlRoot("game")]
	public class GameCenter
	{
		[XmlAttribute("status")]
		public string Status { get; set; }

		[XmlAttribute("id")]
		public string Id { get; set; }

		[XmlElement("venueShort")]
		public string VenueShort { get; set; }

		[XmlElement("venueLong")]
		public string VenueLong { get; set; }

		[XmlElement("wrap")]
		public Wrap Wrap { get; set; }
	}

	public class Wrap
	{
		[XmlElement("mlb")]
		public Mlb Mlb { get; set; }
	}

	public class Mlb
	{
		[XmlElement("headline")]
		public string Headline { get; set; }

		[XmlElement("blurb")]
		public string Blurb { get; set; }
	}
}

File: BaseballTheater/Areas/App/Scripts/Site/MlbDataServer/DataStructures/game_center.ts
namespace Theater
{
	export interface IGameCenter
	{
		status: string;
		id: string;
		venueShort: string;
		venueLong: string;
		wrap: IWrap;
	}

	export interface IWrap
	{
		mlb: IMlb;
	}

	export interface IMlb
	{
		headline: string;
		blurb: string;
		url: IGCUrl;
	}

	export interface IGCUrl
	{
		cid: string;
	}
}